scripts/covid19_india/medical_tests_in_data/preprocess.py

```diff
@@ -0,0 +1,31 @@
+import json
```

Comment:
Could you add an Apache 2.0 License here?

https://github.com/datacommonsorg/data/blob/master/scripts/world_bank/worldbank.py#L1-L13